application/backend/src/main/resources/apps/typerefinery/templates/blank/initial/.content.json

```diff
@@ -3,11 +3,11 @@
   "jcr:content": {
     "jcr:primaryType": "ws:PageContent",
     "sling:resourceType": "typerefinery/components/structure/page",
-    "rootcontainer": {
+    "container": {
       "sling:resourceType": "typerefinery/components/layout/container",
       "jcr:primaryType": "nt:unstructured",
       "decorationTagName": "main",
-      "id": "app"
+      "id": "container"
```

Comment:
we can have no content in here now, let it be blank.

Reply:
as you have already updated body content to just load children or show placeholder.

Reply:
Ok. removed them